### ISSUE TITLE
Error when trying to run the example document on latest Quarto version

### ISSUE THREAD

**Issue by maxdrohde** [ramiromagno/wiley-njd#10]
Hello, thanks for this great Quarto extension! It had been working for me in the past, but when I tried it recently it didn't work with the latest Quarto version 1.4.550. I get this error:

```
ERROR: Template partial partials/abstract.tex was not found. Please confirm that the path to the file is correct.

Stack trace:
    at file:///Applications/quarto/bin/quarto.js:41490:19
    at Array.flatMap (<anonymous>)
    at readPartials (file:///Applications/quarto/bin/quarto.js:41484:20)
    at runPandoc (file:///Applications/quarto/bin/quarto.js:68839:30)
    at eventLoopTick (ext:core/01_core.js:183:11)
    at async renderPandoc (file:///Applications/quarto/bin/quarto.js:70041:26)
    at async Object.onRender (file:///Applications/quarto/bin/quarto.js:77261:36)
    at async renderFileInternal (file:///Applications/quarto/bin/quarto.js:77238:17)
    at async renderFiles (file:///Applications/quarto/bin/quarto.js:77062:17)
    at async render (file:///Applications/quarto/bin/quarto.js:81807:21)
```

I went back and installed Quarto version 1.3.450 and it works on that. Happy to provide more details if it's helpful!

### Quarto Info

```
❯ quarto check
Quarto 1.4.550
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.550
      Path: /Applications/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /Library/TeX/texbin
      Version: 2023

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.11.3
      Path: /Library/Frameworks/Python.framework/Versions/3.11/bin/python3
      Jupyter: 5.3.0
      Kernels: ir, julia-1.6, julia-1.8, julia-1.9, juliapro_v1.5.0-1-1.5, julia-1.5, julia-12-threads-1.8, python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.3.2
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library
      knitr: 1.45
      rmarkdown: 2.25

[✓] Checking Knitr engine render......OK
```

**Comment by maxdrohde** [comment]
I asked about this on the Q&A page and received this feedback:

https://github.com/quarto-dev/quarto-cli/discussions/8796

**Comment by maxdrohde** [comment]
I created a pull request to fix the issue: #11

**Comment by ramiromagno** [comment]
Closed by @maxdrohde in PR https://github.com/ramiromagno/wiley-njd/pull/11. Thank you!
